core/domain/exp_services.py

```diff
@@ -489,6 +489,17 @@ def apply_change_list(exploration_id, change_list):
             elif (
                     change.cmd ==
                     exp_domain.CMD_MIGRATE_STATES_SCHEMA_TO_LATEST_VERSION):
+                # Make sure that the version that you're trying to migrate to
+                # is the latest version.
+                if (
```

Comment:
Did you add a test for this?

(Before you push the PR, make sure that test coverage for the file(s) you're changing still remains at 100%. You can run single tests by following the instructions [here](https://github.com/oppia/oppia/wiki/Running-Tests#server-side-tests), and there are some notes [here](https://github.com/oppia/oppia/wiki/Writing-backend-tests) about measuring coverage.)

Reply:
Thanks - I added two tests for this - one if the migration was successful and one for if it failed in exp_services_test.py. Since there wasn't a test for a successful migration in exp_services_test.py before I wasn't sure where to put it, so I put it after where the other update_exploration blocks are tested. If you want, I'd be happy to move it somewhere else. The code coverage for exp_services.py still isn't 100% but I think it should be a bit better after I added the tests.

Reply:
OK, you'll need to make it 100% though. Find the lines that aren't tested, figure out what behaviour to test that encapsulates those lines of code, and write an appropriate test.